.circleci/config.yml

```diff
@@ -0,0 +1,47 @@
+# Scala CircleCI 2.0 configuration file
+# See: https://circleci.com/docs/2.0/sample-config/
+version: 2
+
+# Define a job to be invoked later in a workflow.
+# See: https://circleci.com/docs/2.0/configuration-reference/#jobs
+jobs:
+  build:
+    # Specify the execution environment. You can specify an image from Dockerhub or use one of our Convenience Images from CircleCI's Developer Hub.
+    # See: https://circleci.com/docs/2.0/configuration-reference/#docker-machine-macos-windows-executor
+    docker:
+      # specify the version you desire here
+      - image: circleci/openjdk:8
+
+      # Specify service dependencies here if necessary
+      # CircleCI maintains a library of pre-built images
+      # documented at https://circleci.com/docs/2.0/circleci-images/
+      # - image: circleci/postgres:9.4
+
+    working_directory: ~/repo
```

Comment:
Does this use shell expansion for this?  I figure you've tested it, but this isn't always a given...

Reply:
This was the default template that CirceCI uses.

Reply:
Good example to show shell expansion is supported :)